src/aws.ts

```diff
@@ -88,22 +89,33 @@ export default {
 
   get s3() {
     return {
-      copyObject: (options: CopyObjectRequest, ignoreMissing = false) =>
+      copyObject: (
+        options: CopyObjectRequest,
+        {dryRun, ignoreMissing, namespace}: {dryRun?: boolean; ignoreMissing?: boolean; namespace?: string},
```

Comment:
do we need to default `ignoreMissing` to `false`?

Reply:
I don't think it's necessary since undefined is falsy